instrumentation/http-url-connection/src/main/java/io/opentelemetry/auto/instrumentation/httpurlconnection/UrlInstrumentation.java

```diff
@@ -96,8 +93,9 @@ public static void errorSpan(
             setPeer(span, host, null);
           }
 
-          DECORATE.onError(span, throwable);
-          span.end();
+          if (throwable != null) {
+            TRACER.endExceptionally(span, throwable);
+          }
```

Comment:
`throwable != null` already checked above
```suggestion
          TRACER.endExceptionally(span, throwable);
```